src/App.tsx

```diff
@@ -1,16 +1,24 @@
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
+import { Movie } from './types/Movie';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => {
+export const App: React.FC = () => {
+  const [visibleMovies, setVisibleMovies] = useState(moviesFromServer);
+
+  const onAdd = (newMovie: Movie) => {
```

Comment:
In the upper component - `handle`, in the down - `on`
```suggestion
  const handleAdd = (newMovie: Movie) => {
```